demo_scripts/exampleAPI.pl

```diff
@@ -195,3 +196,53 @@
 		$marker->right_primer(),
 		$marker->max_primer_dist() );
 }
+
+
+## A9) Find all syntelogues between rice
+
+# get an adaptor to work with comparative sets from compara
+my $mlss_adaptor = Bio::EnsEMBL::Registry->
+	get_adaptor($division, 'compara', 'MethodLinkSpeciesSet');
+
+# find the mlss that describes orthologies between these
+# two rice species
+my $mlss = $mlss_adaptor->fetch_by_method_link_type_registry_aliases(
+    'ENSEMBL_ORTHOLOGUES', ['oryza_sativa', 'oryza_indica']);
+
+# find all homologues between these two rice species
+@homologies = @{$homology_adaptor->
+    fetch_all_by_MethodLinkSpeciesSet($mlss)};
+
+# filter out homologues based on local gene order conservation
+@homologies = grep {
+	$_->goc_score && $_->goc_score >= 75
+} @homologies;
+
+my $n = 1;
+foreach my $homology (@homologies) {
+
+	# get one orthologue
+	my $prot = $homology->get_all_Members->[1];
+
+    # find all orthologues in rice
+    my @rice_homologies = @{$homology_adaptor->
+        fetch_all_by_Member($prot,
+            -METHOD_LINK_TYPE => 'ENSEMBL_ORTHOLOGUES',
+            -TARGET_TAXON => 'Oryza')};

```

Comment:
@brunocontrerasmoreira :Whilst writing this script, I remembered there is the `-TARGET_TAXON` filter, which you could use in the other example to get all the orthologues in _rosids_ ? I should have thought about this earlier ...

Reply:
Thanks, I'll add this to that recipe